lib/definitions/build-strategy.js

```diff
@@ -28,7 +28,8 @@ const DEFAULT_DOCKER_TAG = 'latest';
 module.exports = (options = {}) => {
   // Just doing the source strategy
   const dockerImage = options.dockerImage ? options.dockerImage : DEFAULT_DOCKER_IMAGE;
-  const dockerTag = options.nodeVersion ? options.nodeVersion : DEFAULT_DOCKER_TAG;
+  // remove the options.nodeVersion in 2.0
+  const dockerTag = options.imageTag || options.nodeVersion ? options.imageTag || options.nodeVersion : DEFAULT_DOCKER_TAG;
```

Comment:
Trying to understand the logic here. 

1) If we have `options.imageTag` use it
2) else if we have `options.nodeVersion` use `options.imageTag` (I think this will always be `undefined`)
3) unless `options.imageTag` is `undefined` use `options.nodeVersion` (So I think we'll always end up with `options.nodeVersion`)
4) finally if `options.nodeVersion` is undefined, use `DEFAULT_DOCKER_TAG`

I think all of this can be simplified (and probably should be tested).

```js
const dockerTag = options.imageTag ? options.imageTag : (options.nodeVersion ? options.nodeVersion : DEFAULT_DOCKER_TAG);
```

I.e. if we have `options.imageTag`, use it. Otherwise check if we have `options.nodeVersion` if so, use it, otherwise use `DEFAULT_DOCKER_TAG`. Is there a scenario I am missing here?

Reply:
yea,  i think that is right.   and yea,  that does look more simple.  This will all go away eventually.   This is only really needed when not using the CLI and calling the API directly.

i'll update